lib/rules/string-quotes/README.md

```diff
@@ -75,3 +76,63 @@ a { content: "x"; }
 ```css
 a[id="foo"] {}
 ```
+
+## Optional secondary options
+
+`avoidEscape`: `true|false`, defaults to `true`
+
+Allows strings to use single-quotes or double-quotes so long as the string contains a quote that would have to be escaped otherwise.
+
+### `true`
```

Comment:
I think just one set of examples with `false` will suffice.

They'll need to follow the following format (due to a limitation of how the example background color highlighting works on the website):

---

Allows strings to use single-quotes or double-quotes so long as the string contains a quote that would have to be escaped otherwise.

For example, with `"single", { "avoidEscape" : false }`.

The following patterns are considered violations:

```css
a { content: "x'y'z"; }
```

```css
a[id="foo'bar'baz"] {}
```

The following patterns are *not* considered violations:

```css
a { content: 'x'; }
```

```css
a[id='foo'] {}
```

---